### ISSUE TITLE
Confusing output of tuner.search()

### ISSUE THREAD

**Issue by thepurpleowl** [keras-team/keras-tuner#564]
`Best <objective> So Far` is giving some confusing output.     

For example, in the following code, we can see `Best val_categorical_accuracy So Far` is > 1. Most probably, it's taking the value of `val_loss`. 

But why?

**Model compile code**
```
model.compile(optimizer=keras.optimizers.Adam(learning_rate=hp_learning_rate),
                                                  loss='categorical_crossentropy',
                                                  metrics=['categorical_accuracy'])
```
**Tuner definition**
```
tuner = kt.Hyperband(model_builder,
                     objective=kt.Objective("val_categorical_accuracy", direction="max"),
                     max_epochs=100,
                     factor=3,
                     hyperband_iterations=5,
                     seed=789,
                     directory='my_dir',
                     project_name='intro_to_kt')
```
**Early-stopping** 
```
early_stopping = keras.callbacks.EarlyStopping(
    monitor='val_categorical_crossentropy',
    patience=25,
    min_delta=0.0005
)
```

**tuner.search() statement**
```
tuner.search(X, y, epochs=150, validation_split=0.2, callbacks=[early_stopping])
```
**Output**
```
Trial 3 Complete [00h 00m 28s]
val_categorical_accuracy: 0.35225000977516174

Best val_categorical_accuracy So Far: 1.9555257558822632
Total elapsed time: 00h 01m 27s

Search: Running Trial #4
.
.
.
```

**Comment by haifeng-jin** [comment]
Thanks for the issue!
To summarize, the output log, though says "val_categorical_accuracy", it is actually printing the value of some other metrics.

**Comment by brethvoice** [comment]
@thepurpleowl is this issue still happening? I installed keras-tuner from pip (version 1.1.2) last month and it appears to report the validation categorical accuracy correctly in the console output. Also, it now reports an "Initial best metric for training session" before the start of the first trial run after execution of tuner.search(). This is presumably most useful for tuner searches which are resuming from an existing project folder.
